### PR TITLE
images: Disable PCP units on Fedora images

### PR DESCRIPTION
**Pull request by martinpitt**
These often lead to very long shutdown hangs. The tests already start them on demand, and stop them in various other tests. This is very hard to reproduce at will unfortunately.

There are about 20 srevices and timers, so instead of enumerating them all, disable all `pm*` units.

Fixes #4111

----

 * [x] image-refresh fedora-36
 * [x] image-refresh fedora-37
 * Needs https://github.com/cockpit-project/cockpit/pull/18035

### CODE REVIEW

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/fedora-36-20221213-080000.log

**Comment by cockpituous** [comment]
image-refresh fedora-37 done: https://github.com/cockpit-project/bots/commits/image-refresh-fedora-37-20221213-082850

**Comment by cockpituous** [comment]
Success. Log: https://cockpit-logs.us-east-1.linodeobjects.com/image-refresh-logs/fedora-37-20221213-082828.log

**Comment by martinpitt** [comment]
Seems [f37/machines](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4159-20221213-083341-1520c37b-fedora-37-cockpit-project-cockpit-machines/log.html) became really slow. Retrying to compare.

**Comment by martinpitt** [comment]
[this pixel failure](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4159-20221213-083309-1520c37b-fedora-36-cockpit-project-cockpit/log.html) is a bit irritating -- it seems now the VM runs long enough for the graphs to get a zoom and +/- buttons, which offsets the whole picture. But this [repeats](https://cockpit-logs.us-east-1.linodeobjects.com/pull-4159-20221213-083332-1520c37b-fedora-36-cockpit-project-cockpit/log.html#65).

@mvollmer , @KKoukiou , do you have an opinion what to do there? this feels a bit unpredictable.

**Comment by martinpitt** [comment]
Fixed in https://github.com/cockpit-project/cockpit/pull/18035

File: images/scripts/fedora.setup [hunk 1]
```diff
@@ -200,6 +200,9 @@ rm -rf /var/log/audit/
 # clean up old kernels after upgrading
 dnf repoquery --installonly --latest-limit=-1 -q | xargs --no-run-if-empty rpm -e --verbose
 
+# pmlogger.service sometimes causes long shutdown hangs; disable all PCP services and timers
+systemctl --all --legend=false list-units 'pm*' | awk '{print $1}' | xargs systemctl disable
```
Comment (marusak) [inline images/scripts/fedora.setup:204]: This seems like `pm*` could match also some other unrelated services, but sure, seems fine right now :)

**Review by marusak — APPROVED** [review]
Danke :)

**Pull request merged**